src/main/java/org/broadinstitute/hellbender/tools/picard/sam/ValidateSamFile.java

```diff
@@ -114,7 +114,7 @@ protected Object doWork() {
             validator.setBisulfiteSequenced(IS_BISULFITE_SEQUENCED);
         }
         if (VALIDATE_INDEX) {
-            validator.setValidateIndex(VALIDATE_INDEX);
+            validator.setIndexValidationStringency(BamIndexValidator.IndexValidationStringency.EXHAUSTIVE);
```

Comment:
AFAIK, index validation was binary before, and the concept of stringency level didn't exist. It was introduced by https://github.com/samtools/htsjdk/pull/397. I think the idea was to introduce a less exhaustive and faster validation level. So I think exhaustive is the equivalent of what we used to do. Whether we want to relax that is another question. See https://github.com/broadinstitute/picard/issues/346.